### DISCUSSION TITLE
Trying to add custom colours following the blog post, but there is an error in it that had me stumped. Posting solution here in case it will help others...

### DISCUSSION THREAD

**Discussion by PaulHaze** [saadeghi/daisyui#3100] [General]
So I've been really enjoying playing around with DaisyUI in an Astro project. When googling how to add my own custom colour additions to the theme there was [THIS](https://daisyui.com/blog/how-to-add-new-colors-to-daisyui/) official blogpost. Following along however I could NOT get it to work. 

However if you look closely you will see in the blog they set the variable as an HSL value (without the commas):
```
// importing the built-in 'light' theme
// and setting the color values for '--primary-muted'
// (numbers are HSL values)
...require("daisyui/src/theming/themes")["[data-theme=light]"],
"--primary-muted": "338 83% 66%",
```

yet when they go to use it in Tailwind they treat the variable as OKLCH:

```
    extend: {
      colors: {
        "primary-muted": "oklch(var(--primary-muted) / <alpha-value>)",
      },
    },
```

So to fix it put either ` "primary-muted": "hsl(var(--primary-muted) / <alpha-value>)",` or alternatively use https://oklch.com/ and convert your values into oklch values and use them instead.

Thought I would post it here as it had me stumped for quite some time why it wasn't working...

**Comment by saadeghi** [comment]
Fixed the color values on blog post. Thanks for reporting.
